packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/location.tsp

```diff
@@ -0,0 +1,179 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@resource("locationTrackedResources")
+@parentResource(ResourceGroupLocationResource)
+model LocationTrackedResource is TrackedResource<LocationTrackedResourceProperties> {
+  ...ResourceNameParameter<LocationTrackedResource>;
+}
+
+/** Employee properties */
+model LocationTrackedResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+@armResourceOperations
+interface LocationTrackedResources {
```

Comment:
`LocationResources`

Reply:
Fixed in the new version https://github.com/Azure/cadl-ranch/pull/703/commits/575ac0071e089d23417bcb8d8deb0de15348a1cf